readthedocs/doc_builder/environments.py

```diff
@@ -282,10 +282,10 @@ def save(self):
                     'Content-Type': encoder.content_type,
                 }
             )
-            log.info('Post response via multipart form: %s', resp)
+            log.debug('Post response via multipart form: %s', resp)
         else:
             resp = api_v2.command.post(data)
-            log.info('Post response via JSON encoded data: %s', resp)
+            log.debug('Post response via JSON encoded data: %s', resp)
```

Comment:
These are the most important, as they are currently spamming our build logs.

Reply:
This was added because we were experimenting some issues while posting big outputs. That is working fine now and I think the logs are not more necessary. :+1: on `.debug` it.